internal/website/content/howto/server/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Server"
+date: 2019-06-21T10:36:43-07:00
+draft: true
+showInSidenav: true
+---
+
+The Go CDK's `server` package provides a pre-configured HTTP server with 
+diagnostic hooks for request logging, health checks, and trace exporting via 
+OpenCensus. This guide will show you how to start up and shut down the server,
+as well as how to work with the request logging and health checks.
+
+The Go CDK includes a server package to provide some reasonable defaults, such
```

Comment:
I'm actually not sure we need this paragraph at all. The first paragraph is pretty good now, Curious to hear what other reviewers think here

Reply:
I think the first paragraph is strong enough to stand on its own.